### DISCUSSION TITLE
Kafka Faker

### DISCUSSION THREAD

**Discussion by BenasB** [faker-js/faker#1300] [Show and tell]
Hi folks! First of all, thank you for this project. I'm new to this community, but I just wanted to share how faker inspired me to make an app for producing data to Apache Kafka.

When dealing with apps that use Apache Kafka for messaging, often times I did not have sufficient data for testing purposes and I figured that faker is a great tool to produce that. The outcome is this application: [BenasB/kafka-faker](https://github.com/BenasB/kafka-faker). It has most of faker's functions and has a UI to build a json object using those generation functions. Although there still are some limitations (from my project) to the faker functions used, I'm still happy with the outcome!

![image](https://user-images.githubusercontent.com/29711974/186526069-7d122cda-e2a2-4aa9-974a-e92b4aeec05e.png)

**Comment by xDivisionByZerox** [comment]
Hi @BenasB,
I found this discussion again since we never closed it. Would you like to add your application to our [awesome-faker](https://github.com/faker-js/awesome-faker) repository?

**Reply by BenasB** [reply]
I'm not actively maintaining it, but if that's fine, then go ahead 😄
